packages/geoview-core/src/core/components/feature-info/feature-info.tsx

```diff
@@ -102,6 +106,19 @@ export function FeatureInfo(props: TypeFeatureInfoProps): JSX.Element {
   const [slides, setSlides] = useState<LightBoxSlides[]>([]);
   const [slidesIndex, setSlidesIndex] = useState(0);
 
+  // linkify options
+  const linkifyOptions = {
+    attributes: {
+      title: t('details.external_link'),
+    },
+    defaultProtocol: 'https',
+    format: {
+      url: (value: string) => (value.length > 50 ? `${value.slice(0, 40)}…${value.slice(value.length - 10, value.length)}` : value),
+    },
+    ignoreTags: ['script', 'style'],
```

Comment:
It might be a good idea to also ignore the `img` tag, since the src of img tag might have a path to malicious script to run, just for security purpose, but it depends on your needs.